.github/workflows/post-pull-request-checks-automation.yml

```diff
@@ -40,26 +40,49 @@ on:
         type: number
         default: 1
         description: Configuration for the timeout-minutes parameter of the workflow.
+      run-only-latest-workflow:
+        type: string
+        description: 'Run only if the current workflow is the most recent workflow.'
+        default: 'true'
 
 jobs:
   post-pull-request-checks-automation:
+    outputs:
+      latest-run-number: ${{ steps.get-latest-workflow-run-number.outputs.latest_run_number }}
```

Comment:
Why do we need this output? This is only if you need the job itself to have outputs, not needed for the output of steps.